### DISCUSSION TITLE
Searching in the history...

### DISCUSSION THREAD

**Discussion by FreePDF64** [okaryo/TabTabTab#94] [Ideas]
When searching in TabTabTab, it would be great if it would also search in the history and show the results.

**Comment by okaryo** [comment]
Thanks for your suggestion!
I acknowledge that history contains private records, and accessing it would require user permission. Therefore, I'd like to carefully consider its introduction.
To better understand the feature and the problem it solves, could you please share the circumstances that made you think of including history in the tab search results? What problem do you believe this would solve?

**Comment by FreePDF64** [comment]
Hi,
It often happens that I am looking for a tab, but I closed it a short time ago.

**Reply by okaryo** [reply]
I see, for such cases, we could consider a feature where we save up to 30 recently closed tabs in the extension's internal storage. Then, during a search, we can include these tabs as well, and display them in a category like "Recently Closed" in the search results. This might address your needs.

**Reply by FreePDF64** [reply]
That sounds really nice ;)

**Reply by okaryo** [reply]
In addition to showing them in the search results, it might also be a good idea to add a "Recently Closed" category at the end of the window list.

**Comment by okaryo** [comment]
Issued in #97

**Reply by okaryo** [reply]
done by #252
to be released with v2.22.0
